wiki/Guides/Getting_Your_Map_Modded/th.md

```diff
@@ -0,0 +1,39 @@
+---
+outdated: true
+---
+
+<!-- TODO: modreqs doesn't work anymore -->
+
+# วิธีการทำให้แมพเพลงของคุณถูกตรวจสอบข้อบกพร่อง
+
+ในวิธีการทำให้แมพเพลงของคุณถูกตรวจสอบข้อบกพร่องนั้นเพื่อให้แมพเพลงของคุณถูก[จัดอันดับ](/wiki/Beatmap/Category#ranked)นั้น มันต้องได้รับการ[ตรวจสอบข้อบกพร่อง](/wiki/Modding)ก่อน มีหลายทางที่คุณสามารถขอให้คนตรวจสอบได้ดังนี้
+
+## ช่อง #modreqs
+
+**#modreqs** คือช่องข้อความในตัวเกมที่ๆคุณสามารถขอความช่วยเหลือ([เรื่องตรวจสอบข้อบกพร่อง](/wiki/Modding))ในแมพเพลงของคุณโดยคุณต้องเลือกเพลงที่ต้องการความช่วยเหลือในการหาข้อบกพร่องก่อนใช้คำสั่ง `/np` เพื่อส่งลิ้งค์ไปยังช่องนั้นบางคนที่เป็นคนตรวจสอบข้อบกพร่องหรือบางครั้งก็[Beatmap Nomination Group](/wiki/People/The_Team/Beatmap_Nominators)อาจจะหาแมพเพลงดีๆก็ได้ในลิ้งค์ที่ผู้คนส่งมา
+
+เว็บไซต์จะเก็บคำขอ20อันที่ใหม่ที่สุดและจะเปิดโอกาศให้คนช่วยหาข้อบกพร่องหาแมพเพลง
```

Comment:
A little bit typo. Change `โอกาศ` to `โอกาส`

Reply:
nice